pkgs/development/libraries/science/networking/ns3/default.nix

```diff
@@ -0,0 +1,87 @@
+{ stdenv
+, fetchFromGitHub, fetchurl
+, python
+
+# for binding generation (untested)
+, castxml ? null, pygccxml ? null
+
+# for documentation
+, doxygen ? null, graphviz ? null, imagemagick ? null
+
+# for manual, tetex is used to get the eps2pdf binary
+# texlive to get latexmk. building manual still fails though
+, dia, tetex ? null, ghostscript ? null, texlive ? null
+
+, withDoc ? false
+, withManual ? false
+, withGsl ? false
+
+# --enable-examples
+, withExamples ? false
+
+# --enable-tests
+, withTests ? true
```

Comment:
I would not make testing optional here. This is done by hydra, so most people will not care.

Reply:
Agreed